public/app/features/dashboard/components/PanelEditor/PanelEditorTableView.tsx

```diff
@@ -1,23 +1,43 @@
 import { PanelChrome } from '@grafana/ui';
 import { PanelRenderer } from 'app/features/panel/PanelRenderer';
-import React, { useState } from 'react';
-import { PanelModel } from '../../state';
+import React, { useCallback, useEffect, useState } from 'react';
+import { PanelModel, DashboardModel } from '../../state';
 import { usePanelLatestData } from './usePanelLatestData';
 import { PanelOptions } from 'app/plugins/panel/table/models.gen';
-
+import { RefreshEvent } from 'app/types/events';
+import { Subscription } from 'rxjs';
+import { applyPanelTimeOverrides } from 'app/features/dashboard/utils/panel';
+import { getTimeSrv, TimeSrv } from '../../services/TimeSrv';
 interface Props {
   width: number;
   height: number;
   panel: PanelModel;
+  dashboard: DashboardModel;
 }
 
-export function PanelEditorTableView({ width, height, panel }: Props) {
-  const { data } = usePanelLatestData(panel, { withTransforms: true, withFieldConfig: false }, true);
+export function PanelEditorTableView({ width, height, panel, dashboard }: Props) {
+  const { data } = usePanelLatestData(panel, { withTransforms: true, withFieldConfig: false }, false);
   const [options, setOptions] = useState<PanelOptions>({
     frameIndex: 0,
     showHeader: true,
   });
 
+  const timeSrv: TimeSrv = getTimeSrv();
+  const timeData = applyPanelTimeOverrides(panel, timeSrv.timeRange());
+
+  const onRefresh = useCallback(() => {
+    panel.runAllPanelQueries(dashboard.id, dashboard.getTimezone(), timeData, width);
+  }, [dashboard, panel, timeData, width]);
```

Comment:
Maybe the timeSrv lines should be moved inside the `useCallback`
```suggestion
  const onRefresh = useCallback(() => {
    const timeSrv: TimeSrv = getTimeSrv();
    const timeData = applyPanelTimeOverrides(panel, timeSrv.timeRange());
    panel.runAllPanelQueries(dashboard.id, dashboard.getTimezone(), timeData, width);
  }, [dashboard, panel, width]);
```

Reply:
yup, this makes sense ^^

Reply:
thanks for the suggestion 🙌🏾  https://github.com/grafana/grafana/pull/34998/commits/583d7c4fea42b43c8b4a36202a572ecbec13a234#diff-0b63135356fcc7e14a567cfe8ac6f138a92b792c8622aa2a1aa9e6d845e88337R27 🚀